### PR TITLE
Feature/creator view

### PR DESCRIPTION
**Pull request by nwingt**

### CODE REVIEW

File: src/components/LikeButtonV2/LikeButtonV2.badge.vue [hunk 1]
```diff
@@ -184,10 +195,10 @@ export default {
       };
     },
     shareIconBgColor() {
-      return this.hasSuperLiked ? '#50e3c2' : '#e6e6e6';
+      return this.state === 'shared' ? '#50e3c2' : '#e6e6e6';
     },
     shareIconContentColor() {
-      return this.hasSuperLiked ? '#28646e' : '#9b9b9b';
+      return this.state === 'shared' ? '#28646e' : '#9b9b9b';
```
Comment (nnkken) [inline src/components/LikeButtonV2/LikeButtonV2.badge.vue:201]: Should use `isShared` so less duplication?

File: src/components/LikeButtonV2/LikeButtonV2.vue [hunk 2]
```diff
@@ -343,21 +358,37 @@ export default {
...
   },
   methods: {
     onClick() {
-      this.startClapBitsAnimation();
+      if (
+        ![
+          'unsuperlikeable',
+          'cooldown',
+          'superlikeable-cooldown',
+        ].some(s => s === this.state)
```
Comment (nnkken) [inline src/components/LikeButtonV2/LikeButtonV2.vue:388]: `!['unsuperlikeable', 'cooldown', 'superlikeable-cooldown'].includes(this.state)` seems better

File: src/pages/in/embed/_id/button.vue [hunk 3]
```diff
@@ -140,12 +141,25 @@ export default {
...
         logTrackerEvent(this, 'LikeButtonFlow', 'clickLike', 'clickLike(embed)', 1);
       } else if (this.canSuperLike) {
         await this.newSuperLike();
         await this.updateSuperLikeStatus();
+      } else {
+        if (this.ctaClickCount === undefined) {
+          this.ctaClickCount = 1;
+        } else {
+          this.ctaClickCount = (this.ctaClickCount + 1) % 5;
+          if (this.ctaClickCount !== 0) return;
```
Comment (nnkken) [inline src/pages/in/embed/_id/button.vue:155]: Should it return when `this.ctaClickCount` change from `undefined` to `1`?
If so, prefer some hacks like `this.ctaClickCount = (this.ctaClickCount || 0) + 1` to unify the logic

Comment (nwingt) [reply src/pages/in/embed/_id/button.vue:155]: should not return 1st and every 5th time

Comment (williamchong) [inline src/pages/in/embed/_id/button.vue:155]: Handling civic CTA in `doLike` seems strange and confusing, maybe should split cta part into separated function and call that from here instead

**Review by williamchong — APPROVED** [review]

**Pull request merged**